### DISCUSSION TITLE
Loading discussion dashboard hangs browser

### DISCUSSION THREAD

**Discussion by fnetX** [community/community#3102] [Discussions]
Hello, 
I'm using the latest Firefox on a very low-spec GNU/Linux Debian Buster laptop, and it makes my browser hang for some seconds. Reproducible on every Discussions overview I encountered yet. It renders them unusable in a workflow, I only join discussions when I really want to, since I don't want to wait for my browser to get responsive again.

**Comment by PranamBhat** [comment]
Hello!

Google Chrome is the more efficient and suitable for **GitHub** purpose. Kindly start using Google Chrome!

**Reply by fnetX** [reply]
Sorry, but I'm not going to support the monopoly of the Google Chrome web browser, no way! Even Firefox is becoming uncool, but Google's design decisions regarding the close-down of APIs for their more libre derivates and their force to decide over the direction of the web infrastructure is critical IMO, and no, I use the web browser I like.

If a website is written in a non-efficient manner, the answer is not to change a browser which handles that, but to lighten all the heavy JS on the web, so even older hardware still works with it.
By the way, using chromium is even more laggy on my machine than using Firefox. I don't know how you rate the efficieny of web browsers for specific websites like GitHub.
